quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/PropertyMapper.java

```diff
@@ -260,6 +270,13 @@ public Builder category(ConfigCategory category) {
             return this;
         }
 
+        public Builder type(Class<Boolean> type) {
```

Comment:
this is not used anywhere, is it?

Reply:
not now, please consider it as a baseline for improvements in our config.

Reply:
ok :)